src/api/customers/handlers/get-logged-in-customer.ts

```diff
@@ -21,24 +23,31 @@ export const getLoggedInCustomerQuery = /* GraphQL */ `
     }
   }
 `
-
 export type Customer = NonNullable<GetLoggedInCustomerQuery['customer']>
 
 const getLoggedInCustomer: CustomersHandlers['getLoggedInCustomer'] = async ({
   req,
   res,
   config,
 }) => {
-  const token = req.cookies[config.customerCookie]
+  // use the session if available, otherwise fall back on the customerCookie
+  const session: IronSession = await getIronSession(req, res, sessionOptions)
+  const token: string = session.token || req.cookies[config.customerCookie]
+  const headers: HeadersInit = session.token
+    ? {
+        Authorization: `Bearer ${session.token}`,
```

Comment:
Is BigCommerce API already supported customer impersonation jwt token?

_Request not allowed: Detected that request originated from a browser script and is using a customer impersonation jwt token. Customer impersonation tokens are not allowed with frontend requests._

Reply:
Sorry to ask here as I am working on the same problem and saw this valuable pull request, couldn't figure out where to pass the customer impersonation jwt token I received : )

Reply:
Never mind, including a SHOP_TOKEN cookie in Postman results in 403 Forbidden `Request not allowed`.

Reply:
Yes, it works